src/client/components/global_header/Header.tsx

```diff
@@ -139,17 +139,22 @@ export class Header extends Component<Props, State> {
     }
     const headers = {
       Authorization: token,
+      Accept: 'application/json'
```

Comment:
Gets the auth service to use the Authorization token value from the headers, rather than using cookies.